### PR TITLE
Fixed wikiHow | Added filmmodu and dizilab presence!

### LINKED ISSUE #499: WikiHow only works on english site
**Describe the bug**
WikiHow integration only works in the english site

**To Reproduce**
Just go to the non-english versions of the site

**Expected behavior**
Exactly the same as the english site

**Desktop (please complete the following information):**
 - OS: Ubuntu 19.4
 - Browser: Opera 64.0.3417.54
 - Version: v.2.0.1.3

**Additional context**
Here are all of the wikihow official translations urls:
https://es.wikihow.com/
https://www.wikihow.cz/
https://de.wikihow.com/
https://fr.wikihow.com/
https://hi.wikihow.com/
https://id.wikihow.com/
https://www.wikihow.it/
https://ja.wikihow.com/
https://nl.wikihow.com/
https://pt.wikihow.com/
https://ru.wikihow.com/
https://ar.wikihow.com/
https://th.wikihow.com/
https://www.wikihow.com.tr/
https://www.wikihow.vn/
https://ko.wikihow.com/
https://zh.wikihow.com/

You can check here if I missed any : https://www.wikihow.com/wikiHow:Language-Projects

### LINKED ISSUE #505: Filmmodu | filmmodu.org
**What's the name of the service?**
Ex. https://www.filmmodu.org | Filmmodu

**What should the Presence display?**
its gonna say browsing when browsing
and when ur watching the movie its gonna say what movie are you watching

**Why do you want support for this service?**
cuz this is a good app

**Are you able to provide an image for the service? (512x512 minimum)**
https://ibb.co/w4HHm9Z

### LINKED ISSUE #514: dizilab.
**What's the name of the service?**
Ex. https://dizilab.pw | dizilab.

**What should the Presence display?**
when watching its gonna say watching 
when searching its gonna say searching

**Why do you want support for this service?**
cuz i love this app

**Are you able to provide an image for the service? (512x512 minimum)**
If you can provide the logo pls paste a link in here or the image

### PR DESCRIPTION
**Pull request by eggsywashere**
## dizilab:
_Moving this to the top because it took my 4 hours and it looks SO GOOD k thx._
Alright, now I know that iframes are way more harder than the usual way of creating a presence, well now I also know how to create them. So, next time I'll be the fastest fuc... ;)

~~_Just to note, it looks amazing. I included tons of features. And this closes #514._~~

**Viewing home page:**
homepage

**Viewing other pages:**
other-pages

**Searching for something:**
searching

**Searching for something under a category (category name is inside the brackets):**
searching-with-category

**Viewing a user's profile:**
viewing-a-user-profile

**Viewing an actor's profile:**
viewing-actor-profile

**Viewing a series' profile:**
viewing-series-page

**Watching an episode:**
watching-something

**Watching something from different players/alternatives:**
works-with-different-players

**_Watching something but the video is paused:_**
paused

## filmmodu
filmmodu is a Turkish service that provides latest movies. Oh also, this closes #505.

**Viewing homepage:**
homepage

**Viewing other pages:**
browsing-a-page

**Viewing a category:**
browsing-a-category

**Searching for something:**
searching

**Checking a movie information:**
movie-homepage

**Watching a movie:**
movie-playing

**Watching a movie _but movie is paused this time_:**
movie-paused

## wikiHow Presence Fix:
This was reported in [here](https://github.com/PreMiD/Presences/issues/499) so this update closes #499. I also changed _nearly the whole code_ to make it better. I added a category checker so you can see something different when you're browsing a category.

This is what visually changed but I assure you I've changed a lot:

Also, please note that all language has differs from each other so it is impossible for me to check and get everything work 100% on all other languages. For example, this update makes it work with the Spanish (and all of others) site but Spanish site's author area is different from the main one so it'll show something like "reading _post_ by Informacioné de authòr", not a big problem right?

## TL;DR
`FIXED` wikiHow doesn't work on it's subdomains.
`ADDED` filmmodu presence.
`ADDED` dizilab presence (~~yes, it is amazing~~).

That's all.

### CODE REVIEW

**Comment by eggsywashere** [comment]
Oh, forgot to note that I also added start timestamp so you can see how long you were reading a post _~~AMAZING~~_.
